src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -86,4 +86,38 @@ private[spark] object Config {
       .stringConf
       .checkValue(selectorsValidator, "Selectors must be valid kubernetes labels/selectors")
       .createWithDefaultString(DEFAULT_CLUSTER_SELECTORS)
+
+  private[armada] val singleLabelOrNone: Regex = (s"^($label)?$$").r
+  private[armada] val singleLabelValidator: CharSequence => Boolean = l => {
```

Comment:
This is a method right?  wouldn't it be clearer to define it like so:
```
  private[armada] def singleLabelValidator(l: CharSequence) = {
```

Reply:
Addressed in c796b47